### DISCUSSION TITLE
python setup.py install

### DISCUSSION THREAD

**Discussion by chriskuku** [fail2ban/fail2ban#3592] [General]
Hi. README.md says, use

```
python setup.py install
```

Which python (Ubuntu 22.04)?
Trying ``python3 setup.py install``comes back so fast and silently that I cannot believe anything is being done.

Thanks in advance for helping.

**Comment by sebres** [comment]
> Trying python3 setup.py installcomes back so fast and silently that I cannot believe anything is being done.

Hmm... cannot reproduce this "silent" behavior (here tried with `--dry-run`):
```
$ cd /tmp/fail2ban-1.1
$ sudo python3 setup.py --dry-run install --without-tests
running install
...
running build
...
Copying content of build/scripts-3.11 to /usr/local/bin
```

Note that for previous versions than 1.1 (current master), you would need to invoke 2to3 firstly (the repository was python 2.x based previously).

By the way - here is the wiki how you can install it - https://github.com/fail2ban/fail2ban/wiki/How-to-install-or-upgrade-fail2ban-manually
